frontend/config/moduleFederation.js

```diff
@@ -42,6 +42,11 @@ const readModuleFederationConfigFromPackages = () => {
 };
 
 const getModuleFederationConfig = () => {
+  // Disable module federation for Cypress tests
+  if (process.env.CYPRESS_TESTS === 'true' || process.env.DISABLE_MODULE_FEDERATION === 'true') {
+    return [];
+  }
```

Comment:
Is it necessary to also have `DISABLE_MODULE_FEDERATION` ? Seems redundant.

```suggestion
  // Disable module federation for Cypress tests
  if (process.env.CYPRESS_TESTS === 'true') {
    return [];
  }
```